power_grid_model_c/power_grid_model/include/power_grid_model/common/enum.hpp

```diff
@@ -24,7 +24,7 @@ enum class ControlSide : IntS { from = 0, to = 1, side_1 = 0, side_2 = 1, side_3
 
 enum class CalculationType : IntS { power_flow = 0, state_estimation = 1, short_circuit = 2 };
 
-enum class CalculationSymmetry : IntS { symmetric = 0, asymmetric = 1 };
+enum class CalculationSymmetry : IntS { symmetric = 1, asymmetric = 2 };
```

Comment:
this is ONLY for the C++ core. there SHOULD be no issue, because the `PGM_Options` struct data is translated, not reinterpreted, to its C++ equivalent. Hence, `model.cpp` contains translation logic using a `switch` statement, so that `PGM_asymmetric -> CalculationSymmetry::asymmetric` and `PGM_symmetric -> CalculationSymmetry::symmetric`. NOTE: no `static_cast` or `reinterpret_cast` is used here, so `0` should never be encountered in the C++ core.

Reply:
Let's add that `assert` then, just as a safeguard. Then you can resolve this one.

Reply:
I still do not understand why do we use a different integer value? Can we just sym=1 and aym=0, which is consistent as in the C-API.

Reply:
i have encountered issues because of `0` used for both the default constructor and for actually set values one too many times in my life. my reasoning is that `0` should be `"default"` while anything else needs to be explicitly set. That does align with usage of default initializers.

I will change back for now but I do feel somewhat strongly about things like `power_flow = 0`, `asymmetric = 0`, especially if the default is "bad" (e.g.: most people will use `symmetric` instead of `asymmetric`). There's no going back for the C API, but the C++ core is very much open for any change and I do feel like this would be a good one.

Reply:
okay fine